modules/openshift-developer-cli-installing-odo-on-windows.adoc

```diff
@@ -0,0 +1,29 @@
+// Module included in the following assemblies:
+//
+// * assebmly_installing-odo.adoc     
+
+[id="installing-odo-on-windows"]
+
+= Windows
+
+. Download the latest file from the Bintray link:https://dl.bintray.com/odo/odo/latest/windows-amd64/odo.exe[odo.exe]
+or from the latest release page on link:https://github.com/openshift/odo/releases[GitHub].
+. Extract the file and add the location of extracted binary to your `GOPATH/bin` directory (see below to create a Go binary directory)
+
+== Setting the `PATH` variable for Windows 7/8
```

Comment:
When looking at the TOC for this installation section, it looks odd to me that this path for 7/8 and path for 10 are showing up in the TOC under windows. When all the other install types are saying how to install (using yum or binary), but these are just for setting a path which is needed before you can do what you need to do for windows.

I'd suggest to make these [discrete] so they don't appear in the headings, and maybe add a heading above the steps 1 and 2 above with a description of the Windows method. "Binary installation" I assume it would be here?